pocs/utils/images/__init__.py

```diff
@@ -85,30 +88,45 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
 
 
 def _make_pretty_from_fits(fname, **kwargs):
-    config = load_config()
+    header = getheader(fname)
+    data = getdata(fname)
 
-    title = '{} {}'.format(kwargs.get('title', ''), current_time().isot)
+    title = kwargs.get('title', header.get('FIELD', ''))
+    date_time = header.get('DATE-OBS', current_time()).replace('T', ' ', 1)
```

Comment:
This might need to be tested. `current_time` will return a time object and then the `replace` will probably fail. If you leave at `current_time(pretty=True)` it will return a string and the replace will work on a string (it just won't replace anything since the `T` has already been removed, but that's fine), which is probably safer and means you don't need to bother testing that.

Reply:
I have changed it now but I will test anyways (want to make sure that it doesn't get rid of the T in UTC).

Reply:
Oh, current_time doesn't print out UTC. So it is all good then.